src/components/accounts/CreateAccount.js

```diff
@@ -4,15 +4,17 @@ import { connect } from 'react-redux'
 import CreateAccountForm from './CreateAccountForm'
 import AccountFormSection from './AccountFormSection'
 import AccountFormContainer from './AccountFormContainer'
-import { checkNewAccount, createNewAccount, clear, handleRefreshAccount } from '../../actions/account'
+import { checkNewAccount, createNewAccount, clear, handleRefreshAccount, handleRefreshUrl } from '../../actions/account'
 
 class CreateAccount extends Component {
    state = {
       loader: false,
       accountId: ''
    }
 
-   componentDidMount = () => {}
+   componentDidMount = () => {
+      this.props.handleRefreshUrl(this.props.history.location)
```

Comment:
Isn't this already handle automatically in `Routing.js`?

Reply:
Refactored to remove the ```handleRefreshUrl``` call from ```CreateAccount```.